.bazelci/presubmit.yml

```diff
@@ -18,9 +18,39 @@ tasks:
     build_targets:
     - "//..."
     test_flags:
-    - "--test_tag_filters=-fix-bazelci-ubuntu"
+    - "--test_tag_filters=-e2e,-fix-bazelci-ubuntu"
     test_targets:
     - "//..."
+  ubuntu1604_e2e:
+    name: ubuntu1604_e2e
+    platform: ubuntu1604
+    # We need to reduce the memory & CPU usage of the top-level
+    # bazel process for bazel-in-bazel tests to not
+    # deplete the system memory completely.
+    # - startup JVM memory reduced
+    # - top-level bazel process should use as little memory as possible and only 1 core
+    # - nested bazel process should use a limited number of cores
+    shell_commands:
+    - echo 'startup --host_jvm_args=-Xms256m --host_jvm_args=-Xmx1536m' >> .bazelrc
+    build_targets:
+    - "//e2e/..."
+    test_flags:
+    - "--local_resources=792,1.0,1.0"
+    # test_args will be passed to the nested bazel process
+    - "--test_arg=--local_resources=13288,7.0,1.0"
+    - "--test_arg=--test_tag_filters=-no-bazelci,-no-bazelci-ubuntu"
+    test_targets:
+    - "//e2e/..."
```

Comment:
is it meant to be disjoint with the one above? I was expecting a `test_tag_filters=e2e` to be mutually exclusive with `test_tag_filters=-e2e`

Reply:
👍